README.md

```diff
@@ -1,2 +1,18 @@
 # solutions-engineering-automation
 Repo for automating tasks for Solutions Engineering Team.
+
+
+## How it works
+
+The weekly_tests workflow lists all repos on GitHub maintained by the Solutions Engineering Team, and the job for dispatching test on top of those repos remotely.
+The configuration description:
+ - `repo` is the name of a repository in canonical organization, e.g. charm-apt-mirror
+ - `workflow_file_name` is name of workflow file which should be triggered, e.g. check.yaml
+ - `branch` name of branch from which tests should be executed, e.g. main
+
+Jobs are running in parallel with the maximum number of concurrency defined by the `max-parallel` strategy parameter,
```

Comment:
nit: Would be nice to add a link to the docs page under "max-parallel" with https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#defining-the-maximum-number-of-concurrent-jobs

Reply:
Created #8.